go/platform/apple.bzl

```diff
@@ -0,0 +1,58 @@
+APPLE_GOOS = {
```

Comment:
Need a copyright notice for this file. Just copy from `list.bzl` (but set the year to 2018).